requre/online_replacing.py

```diff
@@ -0,0 +1,72 @@
+import sys
+import re
+import functools
+from typing import List, Callable, Optional, Dict
+
+from requre.storage import PersistentObjectStorage
+from requre.import_system import ReplaceType
+
+def update_module(module_name_filter: str,
+                  replacement: Callable, selector: Optional[str] = "",
```

Comment:
What does it mean to have `selector` as an empty string?

Reply:
in case you have to replace whole module otherwise it will be same as in upgrade import system that there is dot notation to select what to decorate, replace eg "Session.send" in case of empty it will replace whole requrests module